demisto_sdk/commands/unify/unifier.py

```diff
@@ -216,8 +216,17 @@ def get_code_file(self, script_type):
         if self.package_path.endswith('ApiModule'):
             return os.path.join(self.package_path, os.path.basename(os.path.normpath(self.package_path)) + '.py')
 
+        yml_name = os.path.basename(self.yml_path).split('.')[0]
```

Comment:
use `os.path.splitext` in case the integration name contains `.` (like we had `RSA Netwitness 11.1` and was renames due to code not supporting it)